frontend/util.lua

```diff
@@ -792,7 +792,7 @@ function util.findFiles(dir, cb)
             -- lfs can return nil here, as it will follow symlinks!
             local attr = lfs.attributes(path) or {}
             if attr.mode == "directory" then
-                if f ~= "." and f ~= ".." then
+                if not not_recursive and f ~= "." and f ~= ".." then
```

Comment:
Oof :-)

Reply:
Potentially more legible to do something like:

```lua
recursive = type(recursive) ~= "nil" and recursive or false
```

But of course that's uglier a few lines up.